docs/experimental/examples/endpoints_and_collections/create_guest_collection/user_guest_collection.py

```diff
@@ -0,0 +1,45 @@
+import globus_sdk
+from globus_sdk.experimental.globus_app import UserApp
+
+# Tutorial Client ID - <replace this with your own client>
+NATIVE_CLIENT_ID = "61338d24-54d5-408f-a10d-66c06b59f6d2"
```

Comment:
It is a real client ID, so it will work.

(For a long time I resisted having any baked-in available client in the examples, since it's a near-certainty that people will use it instead of registering their own. But it prevented us from having any runnable examples, so I relented.)